google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreOptions.java

```diff
@@ -278,6 +305,12 @@ public void refresh() {}
   protected FirestoreOptions(Builder builder) {
     super(FirestoreFactory.class, FirestoreRpcFactory.class, builder, new FirestoreDefaults());
 
+    if (builder.openTelemetryOptions == null) {
```

Comment:
The design choice was that `FirestoreOptions` will contain a non-null `FirestoreOpenTelemetryOptions`, so that the rest of the codebase doesn't have to check for null. If the `Builder` hasn't set the open-telemetry options, we set the "default" (disabled) open-telemetry options here.

Added some comments.